pkg/test/resourcefixture/testdata/basic/edgecontainer/v1beta1/edgecontainercluster/dependencies.yaml

```diff
@@ -0,0 +1,35 @@
+# Copyright 2023 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+apiVersion: iam.cnrm.cloud.google.com/v1beta1
```

Comment:
If we do not have reference to iam service account, please remove the code here as well

Reply:
I reference the service account in 
```
authorization:
    adminUsers:
      usernameRef:
        name: gsa-${uniqueId}
```
I think this should be the correct way to reference?